app/assets/stylesheets/govuk-component/_govspeak-magna-charta.scss

```diff
@@ -0,0 +1,287 @@
+/*
+ * magna-charta example stylesheet
+ * https://github.com/alphagov/magna-charta
+ *
+ * Author: Tim Paul
+ * Licensed under the MIT license.
+ */
+
+
+// DEFAULT CHART STYLES
+
+$label-width: 40%;          // Horizontal width taken up by the bar labels
+$label-align: right;        // Label alignment
+$base-unit: 16px;           // For font sizes and related spacing
+$compact-font-size: 12px;   // For the 'compact' bar style
+$bar-padding: 8px;          // The padding between the bar values and the edge of the bar
+$bar-spacing: 5px;          // The spacing around the bars
+$stack-spacing: 1px;        // For stacked bars, the spacing between each stack in a bar
+$caption-side: top;         // Caption alignment. Top or bottom.
+$key-width: 160px;           // Only used by IE. Other browsers get smallest width that fits content
```

Comment:
Would it be possible to move the variables inside the `.govuk-govspeak` selector. That will force the variables to be scoped so they can't accidentally be used by anything else.